app/src/main/java/com/mathroule/demoapp/MainActivity.java

```diff
@@ -28,6 +30,22 @@ public void onClick(View view) {
                         .setAction("Action", null).show();
             }
         });
+
+        try {
+            String text = Helper.getRandomButtonLabel();
+            ((Button) findViewById(R.id.buton)).setText(text);
+
+            if (text == "something") {
+                ((Button) findViewById(R.id.buton)).setText("something");
+            }
+        } catch (Exception e) {
+        }
+        findViewById(R.id.buton).setOnClickListener(new OnClickListener() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mathroule/demo-app/blob/6ba8726f5796dfc005c0166d5e5fe29dcee50c6b/app/src/main/java/com/mathroule/demoapp/MainActivity.java#L43"></span>The class com.mathroule.demoapp.MainActivity$2 could be refactored into a named <em>static</em> inner class</td>
    </tr>
  </tbody>
</table>